dice_roller_cog.py

```diff
@@ -47,21 +49,16 @@ def __init__(self, bot):
     @option("dc", description="Number to which dice result will be compared", required=False)
     async def post(self, ctx: discord.ApplicationContext, roll: str, dc: int = None, secret: str = "Open"):
         try:
+            # print('Rolling')
             engine = get_asyncio_db_engine(user=USERNAME, password=PASSWORD, host=HOSTNAME, port=PORT, db=SERVER_DATA)
-            async_session = sessionmaker(engine, expire_on_commit=False, class_=AsyncSession)
-            async with async_session() as session:
-                result = await session.execute(
-                    select(Global).where(
-                        or_(
-                            Global.tracker_channel == ctx.interaction.channel_id,
-                            Global.gm_tracker_channel == ctx.interaction.channel_id,
-                        )
-                    )
-                )
-                guild = result.scalar()
+            guild = await initiative.get_guild(ctx, None)
             try:
                 roll_result = d20.roll(roll)
-                roll_str = opposed_roll(roll_result, d20.roll(dc)) if dc else roll_result
+                # print(roll_result)
+
+
+                roll_str = opposed_roll(roll_result, d20.roll(f"{dc}")) if dc else roll_result
+                # print(f"roll_str: {roll_str}")
```

Comment:
```suggestion
                roll_str = opposed_roll(roll_result, d20.roll(f"{dc}")) if dc else roll_result
                # print(f"{roll_result =} {roll_str =}")
```
Just a simple reducing needless whitespace. Also a fancy trick for printing a variable name and it's value all at once.